pkg/smc/consts_arm64.go

```diff
@@ -25,4 +33,10 @@ var allKeys = []string{
 	AdapterKey2,
 	AdapterKey3,
 	BatteryChargeKey,
+	DcinCurrentKey,
```

Comment:
```suggestion
	DCInCurrentKey,
```

Go's code style is a bit different: Dcin -> DCIn